Assets/__Scripts/Extensions/BeatSaberSongExtensions.cs

```diff
@@ -140,6 +140,17 @@ public static Dictionary<string, string> GetFilesForArchiving(BaseInfo info)
             TryAddToFileDictionary(exportedFiles, info.Directory, map.LightshowFileName);
         }
 
+        // while we could just add the specific bookmark files for each diff, for better official editor compatibility it makes more sense to add every bookmark file that exists
+        if (info.MajorVersion == 4)
+        {
+            var bookmarkFiles = Directory.GetFiles(Path.Combine(info.Directory, "Bookmarks"), "*.bookmarks.dat");
```

Comment:
Check if the directory exists. This fails on official editor maps without any bookmarks.